src/Microsoft.VisualStudio.Services.Agent/AgentServer.cs

```diff
@@ -28,6 +31,9 @@ public interface IAgentServer : IAgentService
         // job request
         Task<TaskAgentJobRequest> RenewAgentRequestAsync(int poolId, long requestId, Guid lockToken, CancellationToken cancellationToken = default(CancellationToken));
         Task<TaskAgentJobRequest> FinishAgentRequestAsync(int poolId, long requestId, Guid lockToken, DateTime finishTime, TaskResult result, CancellationToken cancellationToken = default(CancellationToken));
+
+        // ReleaseManagement
+        IEnumerable<AgentArtifactDefinition> GetReleaseArtifactsFromService(Guid teamProject, int releaseId, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Agree. We will have more methods in RM server also going forward. Good to have it in a separate server class.